app/sitemap.ts

```diff
@@ -19,7 +19,10 @@ export default async function sitemap(): Promise<MetadataRoute.Sitemap> {
     cv: Date.now(),
   });
 
-  const ret = response.map((story) => {
+  // Exclude any stories with noindex set to true on Storyblok
+  const indexStories = response.filter((story) => !story.content?.noindex);
```

Comment:
exclude any stories with noindex=true in Storyblok

Reply:
The label in SB is a little misleading IMHO. It doesn't block search indexing but rather, removes it from the sitemap so the only way it would show up in google is by it being crawled. If you want to block the page from being indexed, you need to set a header or meta tag to tell google not to index that path.

Reply:
Hi @sherakama, I actually did add the meta tag for noindex previously in the getPageMetadata utility here:
https://github.com/SU-SWS/ood-giving-campaign/blob/04f9a3a3fd194d69a5791c6991fb2f6ba1acf524/utilities/getPageMetadata.ts#L99

![Screenshot 2024-01-04 at 11 21 32 PM](https://github.com/SU-SWS/ood-giving-campaign/assets/42749717/153b6de2-4eaf-4d66-a476-c5f07586bd86)


In this PR, I decided to also exclude those noindex pages in the sitemap to avoid giving search engines conflicting info.

Reply:
Ah, that makes sense. Thank you for clarifying.